CoreService/Workflow.psm1

```diff
@@ -0,0 +1,231 @@
+	
+	.Example
+    Get-TridionWorkflowItems -AssignedToById "tcm:0-1014-65552"
+	Get-TridionWorkflowItems -AssignedToById "tcm:0-15-65568"
+	Returns a list of all workflow items within Tridion filtered by AssignedTo given ID of User or Group.
+	
+	.Example
+    Get-TridionWorkflowItems -AssignedToByTitle "TRIDIONDEV\San"
+	Get-TridionWorkflowItems -AssignedToByTitle "Developer"
+	Returns a list of all workflow items within Tridion filtered by AssignedTo given Title of User or Group.
+	
+	.Example
+    Get-TridionWorkflowItems -Status "Active" | Select-Object Title, Id
+	Get-TridionWorkflowItems -AssignedToById "tcm:0-1014-65552" | Select-Object Id,Title
+	Returns a list of the Title, Id of all workflow items within Tridion filtered By Status(Active) or AssignedTo filter of The Workflow.
+    #>
+    [CmdletBinding(DefaultParameterSetName='Status')]
+    Param
+    (
+		[Parameter(ParameterSetName='Status')]
+		[ValidateSet('', 'Active', 'Historical')]
+		[string]$Status,
+		
+		# The TCM URI of the Publication Target to load.
+        [Parameter(ParameterSetName='AssignedToById')]
+		[ValidateNotNullOrEmpty()]
+        [string]$AssignedToById,
+		
+		# The Title of the Publication Target to load. This is slower than specifying the ID.
+        [Parameter(ParameterSetName='AssignedToByTitle')]
+		[ValidateNotNullOrEmpty()]
+        [string]$AssignedToByTitle
+	)		
+	
+	Begin
+	{
+        $client = Get-TridionCoreServiceClient -Verbose:($PSBoundParameters['Verbose'] -eq $true);
+	}
+	
+    Process
+    {
+		$filter = New-Object Tridion.ContentManager.CoreService.Client.ProcessesFilterData;
+		$acvities = new-object Tridion.ContentManager.CoreService.Client.ActivityData;
+		$readOption = New-Object Tridion.ContentManager.CoreService.Client.ReadOptions;
+		$readOption.LoadFlags = [Tridion.ContentManager.CoreService.Client.LoadFlags]::None;
+		$processInstanceData = New-Object Tridion.ContentManager.CoreService.Client.ProcessInstanceData;
+		$list = New-Object -TypeName System.Collections.Generic.List[Tridion.ContentManager.CoreService.Client.ProcessInstanceData];
+					
+        if ($client -ne $null)
+        {
+			switch($PsCmdlet.ParameterSetName)
+			{
+			    'Status'
+				{
+					Write-Verbose "Loading list of Workflow Items...";
+					
+					if($Status -eq $null)
+					{
+						$filter.ProcessType = [Tridion.ContentManager.CoreService.Client.ProcessType]::Any;				
+					}
+					if($Status -eq "Active")
+					{
+						$filter.ProcessType = [Tridion.ContentManager.CoreService.Client.ProcessType]::Active;				
+					}
+					if($Status -eq "Historical")
+					{
+						$filter.ProcessType = [Tridion.ContentManager.CoreService.Client.ProcessType]::Historical;				
+					}			
+								
+					$WorkflowItemDetail = $client.GetSystemWideList($filter);					
+					return $WorkflowItemDetail
+				}
+			    'AssignedToById'
+				{
+					Write-Verbose "Loading Publication Target with Id '$AssignedToById'..."
+					if ($AssignedToById -ne $null)
```

Comment:
I'm not sure it's worth it to actually load the user/group and validate that it exists etc.
It won't perform very well and in the end if you filter on the wrong ID you should probably just get nothing back.